### PR TITLE
#16: Implement key-value display view

### PR DESCRIPTION
**Pull request by olena-syrotenko**
Created UI to display key-value pairs. It has an empty space on the right to display the folders tree. Also, need to set up navigation between canvases.
Snapshot_data
Snapshot_empty

### CODE REVIEW

**Comment by olena-syrotenko** [comment]
As a navigation solution we can use a Frame object instead of a Canvas

**Comment by syrotenko** [comment]
> It has an empty space on the right to display the folders tree

On the left =)

> Also, need to set up navigation between canvases

Yes, we can implement navigation but let's simplify it now. Propose to not hide elements for connection and implement navigation improvement it separate issue.

File: app_view.py [hunk 1]
```diff
@@ -64,3 +65,29 @@ def connect_click(self):
...
+
+        table = ttk.Treeview(canvas, yscrollcommand=table_scroll.set)
+        table.grid(column=1, columnspan=2,  row=0, sticky="nsew")
+        table.config(height=15)
+        table_scroll.config(command=table.yview)
+
+        table['columns'] = ('Key', 'Value')
+
+        table.column("#0", width=0, stretch=NO)
+        table.column("Key", width=200, anchor=CENTER)
```
Comment (syrotenko) [inline app_view.py:87]: Let's use `tk.CENTER` and do not import all items from `tkinter.constants`

**Review by syrotenko — CHANGES_REQUESTED** [review]

**Comment by syrotenko** [comment]
Also, how do we pass the data to the tree view? Can we pass the data after the tree view was created?

File: app_view.py (hunk 1 above)
Comment (olena-syrotenko) [reply app_view.py]: done

**Comment by olena-syrotenko** [comment]
> Also, how do we pass the data to the tree view? Can we pass the data after the tree view was created?
We can pass the data to the tree view using `insert()` function, for example: `table.insert(parent='',index='end',text='', values=('Some key','Some pass','101'))`. So we can update the tree view content on a specific event.

**Review by syrotenko — APPROVED** [review]

**Pull request merged**